client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -6,18 +6,17 @@ import useStyles from './useStyles';
 interface Props {
   linkTo: string;
   btnText: string;
+  variant: any;
 }
 
-const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText, variant }: Props): JSX.Element => {
```

Comment:
Please rename this component to give it a more suitable name, as it renders only a Button.